### ISSUE TITLE
Trouble importing Ink files, unable to create an InkpotStoryAsset

### ISSUE THREAD

**Issue by DorminVR** [The-Chinese-Room/InkpotDemo#36]
First off, thanks so much for releasing this to the world! Very excited to see more support for Ink in Unreal. 

I've followed the setup instructions and gotten the demo running, I am able to play the Demo and HelloInk stories within the editor. I am having a trouble importing my own ink files though, when I drag ink files into the content browser nothing happens? I've also tried importing ink files by right clicking and importing them through a file browser. 

I've tested this with my [own ink files](https://drive.google.com/drive/folders/19Uds47xxxV5VI7-n3-HVllUy6N4xeSks?usp=sharing) and [ink test files](https://github.com/inkle/ink/tree/master/tests) provided by Inkle and I'm having the same issue. Any ideas? 

P.S. Does this plugin support Simplified Chinese characters? Dave's UnrealInk plugin didn't support Chinese characters or Cyrillic script, so I wanted to test out this plugin to see if it would be a viable option for a translation of a game I released. You all are amazing for sharing your hard work, we appreciate it!

**Comment by TCR-Nick** [comment]
Hi @DorminVR 

Check the output log and see if there are any warnings.

_Inkpot: Warning: CompileInkString - 
InkpotDemo/Plugins/Inkpot/Intermediate/InkCommandLine/1/2/TempInkFile.ink failed to compile ink string. Error 
You must install or update .NET to run this application._

The Ink compiler, InkleCate,  included with Inkpot needs .net 3.1 installed. 
Without this the ink scripts will not compile and no asset will be created in the UE content browser.

To fix, install the 3.1 .net framework from here
https://dotnet.microsoft.com/en-us/download/dotnet/3.1/runtime?cid=getdotnetcore

**Comment by TCR-Nick** [comment]
I managed to import both your files, here's a screen shot from your simplified chinese script.

![image](https://github.com/The-Chinese-Room/InkpotDemo/assets/141139646/f5d2edd6-eb06-4d1a-af1e-f8a283b116b3)

**Comment by DorminVR** [comment]
Thanks for confirming the ink files are working on your end. I already had Net 3.1 installed but wasn't aware of the output log, so ty for pointing that out. After checking the output log it looks like the plugin is having trouble finding the inkcate.exe. 

![InkPotError](https://github.com/The-Chinese-Room/InkpotDemo/assets/69361026/627e5f86-9489-43eb-ad4d-83957f38d64a)

I've verified the inklecate.exe is within the project, so I'm not sure why it's having trouble finding it. Any ideas?

![image](https://github.com/The-Chinese-Room/InkpotDemo/assets/69361026/7e54006f-f4a2-4a20-8f04-6dbb74d0f8f8)

Edit, I got the import working. Very much a user error, didn't realize I had screwed up the folders for my install, had a folder called Inkpot and Inkpot-release under my plugins folder. Moving the contents from Inkpot-release into the Inkpot folder fixed the issue for me. Thanks for the swift reply, this plugin is incredible!
